lib/screens/authentication/widgets/verify_button.dart

```diff
@@ -1,14 +1,218 @@
 import 'package:flutter/material.dart';
+import 'package:saasify/screens/onboarding/companies_screen.dart';
 
+import '../../../data/customer_cache/customer_cache.dart';
+import '../../../data/models/authentication/authentication_model.dart';
+import '../../../di/app_module.dart';
 import '../../../utils/constants/string_constants.dart';
 import '../../../widgets/primary_button.dart';
+import '../../../widgets/sidebar.dart';
 
 class AuthVerifyButton extends StatelessWidget {
-  const AuthVerifyButton({super.key});
+  final CustomerCache _customerCache = getIt<CustomerCache>();
+
+  AuthVerifyButton({super.key});
 
   @override
   Widget build(BuildContext context) {
     return PrimaryButton(
-        onPressed: () {}, buttonTitle: StringConstants.kVerify);
+        onPressed: () {
+          _customerCache.setIsLoggedIn(true);
```

Comment:
this logic should be in bloc and not in the ui

Reply:
yes it is already implemented in authentication bloc but as ajay was working on authentication functionality so i had to hardcode it in the ui and I kept it so that you could see the modified UI .